src/lib/viewers/image/ImageControls.tsx

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
 import AnnotationsControls, { Props as AnnotationsControlsProps } from '../controls/annotations';
+import ColorPickerControl, { Props as ColorPickerControlProps } from '../controls/color-picker';
 import ControlsBar from '../controls/controls-bar';
 import FullscreenToggle, { Props as FullscreenToggleProps } from '../controls/fullscreen';
 import RotateControl, { Props as RotateControlProps } from '../controls/rotate';
 import ZoomControls, { Props as ZoomControlsProps } from '../controls/zoom';
+import { AnnotationColor } from '../../AnnotationModule';
+import { AnnotationMode } from '../controls/annotations/types';
 
-export type Props = AnnotationsControlsProps & FullscreenToggleProps & RotateControlProps & ZoomControlsProps;
+const colors = Object.values(AnnotationColor);
+
+export type Props = AnnotationsControlsProps &
+    ColorPickerControlProps &
```

Comment:
Are any of these props passed to ImageControls? If not, is this needed?

Reply:
Not needed, `annotationColor` is already being imported by AnnotationsControls.